File: crypto/simple-otp/solve.py
import random

encoded_with_xor = b'\x81Nx\x9b\xea)\xe4\x11\xc5 e\xbb\xcdR\xb7\x8f:\xf8\x8bJ\x15\x0e.n\\-/4\x91\xdcN\x8a'

random.seed(0)
key = random.randbytes(32)

assert len(key) == len(encoded_with_xor)

print(''.join(map(chr, (key[i] ^ encoded_with_xor[i] for i in range(len(key))))))

File: crypto/pope-shuffle/solve.js
let s = `࠴࠱࠼ࠫ࠼࠮ࡣࡋࡍࠨ࡛ࡍ࡚ࡇ࡛ࠩࡔࡉࡌࡥ`;
let ord = c => c.codePointAt(0);
let chr = i => String.fromCodePoint(i);
console.log([...s].map(c=>chr(ord(c)-ord(s.at(0))+ord("L"))).join(""));
